project/Version.scala

```diff
@@ -0,0 +1,18 @@
+case class Version(major: Int, minor: Int, patch: Int) {
```

Comment:
@julienrf alternatively, we could use scalacenter/versions if you merge https://github.com/scalacenter/versions/pull/3 and push a tag.